### DISCUSSION TITLE
How do you manage default apps?

### DISCUSSION THREAD

**Discussion by mightyiam** [gvolpe/nix-config#7] [Q&A] [answered]
Would you mind sharing? For example, my `xdg-open http://example.com` opens Chromium, which I'd like it to open Firefox, instead.

**Comment by mightyiam** [comment, marked as answer]
Found an example here:
https://github.com/nix-community/home-manager/issues/1028

**Reply by gvolpe** [reply]
I only use Firefox so that's set up for me without extra configuration but if you have multiple browsers then `xdg` is where you need to look at if you're running on Xorg.
